### DISCUSSION TITLE
[ FF7] 7th Heaven crash when using NT 2.0

### DISCUSSION THREAD

**Discussion by FtzofRage** [julianxhokaxhiu/FFNx#266] [Q&A] [answered]
While using 7th heaven the New Threat 2.0 causes the game to crash when starting. 7th heaven version is v2.3.1.66
![F5F9EC26-76D1-4D72-8E87-477DAF1D3E90](https://user-images.githubusercontent.com/86936437/124405140-cede8380-dd0b-11eb-9c43-79594934409e.jpeg)

![B16D53E3-D2AA-4AEC-8F1F-4B2A7D9BE866](https://user-images.githubusercontent.com/86936437/124405087-98086d80-dd0b-11eb-9c2e-059b86ad2dbe.jpeg)
**PLEASE NOTE:** If you come here to report a crash while you were using 7th Heaven, please do on their relative support channels. NO SUPPORT will be provided, and your issue will be close immediately.

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**GPU (please complete the following information):**

- Brand [e.g. Nvidia]
- Driver Version [e.g. 27.21.14.5730]

**Additional context**

**Comment by wturner859** [comment, marked as answer]
This is a 7th heaven issue. Not an FFnx issue. Stick with the 7th Heaven build Chris gave you until it's fixed.
